### PR TITLE
Update datafile columns

### PR DESCRIPTION
**Pull request by veidenberg**
### Description
Update GB code with the new datafile structure (column order) for repeat- and simple feature tracks.

### Related JIRA Issue(s)
[ENSWBSITES-2576](https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-2576)

### Review App URL(s) 
http://new-datafiles.review.ensembl.org/genome-browser/grch38?focus=gene:ENSG00000221914

### Data Availability

New GB datafiles (for MVP) are copied to the NFS dev area and Track API is updated for testing in review apps. 

### Dependencies 
Before merging this PR: 
- Datafiles need to me synced to staging/prod area
- Track API needs to be udpated in staging (and later in prod)

### CODE REVIEW

File: backend-server/app/data/v16/simplefeatures.py [hunk 1]
```diff
@@ -13,24 +13,21 @@ def get_features(
     ends = []
     strands = []
     analyses = []
-    scores = []
 
     for (start, end, rest) in data:
-        (strand, analysis, score) = rest.split("\t")
+        (strand, analysis) = rest.split("\t")
         chrs.append(chrom.name)
```
Comment (kamaldodiya) [inline backend-server/app/data/v16/simplefeatures.py:19]: Why not take this out of loop and do something like 
```
chrs = [ chrome.name ] * len(data)
```

**Review by kamaldodiya — COMMENTED** [review]
Looks good just one comment to intialise chrs outside loop as it doesn't change

**Review by azangru — APPROVED** [review]
Checked the diff and the deployment; both look reasonable.

**Comment by kamaldodiya** [comment]
The comment I made is existing code and not part of this PR. Looks good to me.

**Review by kamaldodiya — APPROVED** [review]

**Pull request merged**